tests/tracing/test_sampling.py

```diff
@@ -284,3 +284,30 @@ def test_warns_and_sets_sampled_to_false_on_invalid_traces_sampler_return_value(
         transaction = start_transaction(name="dogpark")
         logger.warning.assert_any_call(StringContaining("Given sample rate is invalid"))
         assert transaction.sampled is False
+
+
+@pytest.mark.parametrize("traces_sample_rate", [None, 0.0, 1.0])
```

Comment:
Do we have tests that cover the `traces_sampler` equivalent side of this?

Reply:
will add another one, basically the same logic